common/src/main/java/com/wynntils/features/internal/FixPacketBugsFeature.java

```diff
@@ -19,12 +21,11 @@
 import net.minecraftforge.eventbus.api.EventPriority;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-public class FixPacketBugsFeature extends FeatureBase {
-    private static final int METHOD_ADD = 0;
+@EventListener
+@StartEnabled
+public class FixPacketBugsFeature extends InternalFeature {
 
-    public FixPacketBugsFeature() {
-        setupEventListener();
-    }
+    private static final int METHOD_ADD = 0;
```

Comment:
What's this? O_o

Reply:
it seems to be a quirk, the line was present originally

Reply:
Ah, I see. The line is unchanged. Just the diff looked weird. (Now I recognize it; I added it originally 8-))